src/amg/azext_amg/custom.py

```diff
@@ -297,6 +297,36 @@ def restore_grafana(cmd, grafana_name, archive_file, components=None, remap_data
             destination_datasources=data_sources)
 
 
+def migrate_grafana(cmd, grafana_name, source_instance_ip, source_instance_token, dry_run=False, overwrite=False, folders_to_include=None,
```

Comment:
source_endpoint?

Reply:
Yeah, that sounds good, but I want to take advantage of -n flag to be consistent with other commands. How would I implement that?